e2e/tests/steps_init_test.go

```diff
@@ -23,8 +23,8 @@ import (
 var e2eLogger = pocketLogger.Global.CreateLoggerForModule("e2e")
 
 const (
-	// defines the host & port scheme that LocalNet uses for naming validators.
-	// e.g. validator-001 thru validator-999
+	// Each actor is represented e.g. validator-001 thru validator-999
+	// Defines the host & port scheme that LocalNet uses for naming actors.
 	validatorServiceURLTmpl = "validator-%s-pocket:%d"
```

Comment:
The comment doesn't seem to match the pattern it's describing.

```suggestion
	// Each actor is represented e.g. validator-001-pocket:42069 thru validator-999-pocket:42069.
	// Defines the host & port scheme that LocalNet uses for naming actors.
	validatorServiceURLTmpl = "validator-%s-pocket:%d"
```